README.md

```diff
@@ -11,22 +11,22 @@ Watch from minute 21 to 41 of the [September 1st Jupyter meeting video recording
 * A base extension that enable the use of [Web Components](http://webcomponents.org) and [Polymer](https://www.polymer-project.org/1.0/) elements
 * A set of core elements facilitate interacting with code running on the kernel
 * Extensions to data binding support and installing of 3rd party elements.
-* Implementations for Python Kernel and Scala using [Apache Toree](https://github.com/apache/incubator-toree)
+* Implementations for Python, R (using [IRkernel](https://github.com/IRkernel/IRkernel)) and Scala (using [Apache Toree](https://github.com/apache/incubator-toree))
 
 ## What It Lacks
 
-* Support for state persistance and downstream tools (nbviewer)
+* Support for disconnected (no kernel) environments (i.e. nbviewer)
 * Interactions with DataFrames. Currently read-only.
 
 ## Runtime Requirements
 
-* Jupyter Notebook 4.0.x running on Python 3.x or Python 2.7.x (see the [0.1.x branch](https://github.com/jupyter-incubator/declarativewidgets/tree/0.1.x) for IPython Notebook 3.2.x compatibility)
-* [IPywidgets](https://github.com/ipython/ipywidgets) 4.1.x
+* Jupyter Notebook 4.0.x, 4.1.x, or 4.2.x running on Python 3.x or Python 2.7.x (see the [0.1.x branch](https://github.com/jupyter-incubator/declarativewidgets/tree/0.1.x) for IPython Notebook 3.2.x compatibility)
+* [IPywidgets](https://github.com/ipython/ipywidgets) 4.1.x and 5.0.1+ (R and Scala support not available for 5.0.0)
```

Comment:
Make sure you update for 5.0.1 or 5.1.0 version number choice when @SylvainCorlay cuts the release